### ISSUE TITLE
#ref in additional properties

### ISSUE THREAD

**Issue by fiddlededee** [fmvilas/openapi3-generator#27]
Hi. I try to make a template for asciidoc (asciidoctor, in fact). With test examples everythin is great. But I faced the following problem.

Here is the part of specification:
```
    ModelMetaData:
      required:
        - members
        - attributes
      properties:
        members:
          description:
            Meta data for members
          additionalProperties: # key - memberId
            $ref: '#/components/schemas/MemberMetaData'
        attributes:
          description:
            Meta data for attributes
          additionalProperties: # key - attributeId
            $ref: '#/components/schemas/AttributeMetaData'

    MemberMetaData:
      required:
        - text
      properties:
        text:
          description:
            id of the member as a text
          type: string
        aggregateText:
          description:
            aggregate of the member as a text
          type: string
        formulaeText:
          description:
            formulae of the member as a text
          type: string

    AttributeMetaData:
      required:
        - text
      properties:
        text:
          description:
            id of the attribute as a text
          type: string
```

In the resulting table I get summaryAsHTML, descriptionAsHTML and generatedExample as extra rows and get empty descriptions. 

![image](https://user-images.githubusercontent.com/34554890/61576048-3ed2af80-aadd-11e9-84f0-34a465c8952a.png)

The same problem is with Markdown template. 

Here is the template code, that iterates through properties:

```
{{#each prop.additionalProperties.properties}}
{{> schemaProp prop=. propName=@key required=(isRequired ../prop.additionalProperties @key) path=(buildPath ../propName ../path @key)}}
{{/each}}
```

If the problem can be fixed in a template, just give a hint, I'll make PR for Markdown and if you don't mind I can make PR for asciidoc template.

**Comment by fiddlededee** [comment]
Probably, solved this in #28

**Comment by fmvilas** [comment]
Fixed in #28.
